package.json

```diff
@@ -17,6 +17,7 @@
     "release:create": "create-release",
     "release": "./scripts/publish-release.sh",
     "start:playground": "node test/app.js",
+    "start:playground:selfContained": "PORT=3031 SELF_CONTAINED=true npm run start:playground",
```

Comment:
Maybe `start:playground:scs` (where "scs" stands for "self-contained styles")?

Reply:
so much better! thank you